hooks/useCreateProposal.ts

```diff
@@ -88,6 +88,78 @@ export default function useCreateProposal() {
       selectedGovernance.account.proposalCount,
       instructionsData,
       isDraft,
+      false,
+      ["Approve"],
+      client
+    )
+    queryClient.invalidateQueries({
+      queryKey: proposalQueryKeys.all(connection.endpoint),
+    })
+    return proposalAddress
+  }
+
+  /** @deprecated because the api is goofy, use `proposeMultiChoice` */
+  const handleCreateMultiChoiceProposal = async ({
```

Comment:
just make `proposeMultiChoice` the only function, no reason for this one to exist. `handleCreateProposal` still exists because it gets used elsewhere and I did not bother to refactor.